torch/csrc/jit/backends/backend_init.cpp

```diff
@@ -0,0 +1,126 @@
+#include <torch/csrc/jit/backends/backend_init.h>
+#include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/runtime/custom_operator.h>
+
+namespace torch {
+namespace jit {
+void initJitBackendBindings(PyObject* module) {
+  for (auto& backend : getAllBackends()) {
```

Comment:
What if a backend gets registered after this is called? This will be the common case because backends will likely be in different modules from `torch`.